pkgs/by-name/tu/tusd/package.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, buildGoModule, fetchFromGitHub, nixosTests }:
+
+buildGoModule rec {
+  pname = "tusd";
+  version = "2.8.0";
+
+  src = fetchFromGitHub {
+    owner = "tus";
+    repo = "tusd";
+    rev = "v${version}";
+    hash = "sha256-OzXBeLDjaJk4NVgsauR/NUATh7qHbuEfWNdhytZmd0A=";
+  };
+
+  vendorHash = "sha256-YununGyB72zE0tmqO3BREJeMTjCuy/1fhPHC5r8OLjg=";
+
+  # Tests need the path to the binary:
+  # https://github.com/tus/tusd/blob/0e52ad650abed02ec961353bb0c3c8bc36650d2c/internal/e2e/e2e_test.go#L37
+  preCheck = ''
+    export TUSD_BINARY=$PWD/../go/bin/tusd
+  '';
+
+  passthru.tests.tusd = nixosTests.tusd;
+
+  meta = with lib; {
+    description = "Reference server implementation in Go of tus: the open protocol for resumable file uploads";
+    license = licenses.mit;
```

Comment:
Please remove `with lib;` and use `lib.licenses.mit and `with lib.maintainers;`

Reply:
Done.